dft/src/butterflies.rs

```diff
@@ -57,17 +103,39 @@ pub trait Butterfly<F: Field>: Copy + Send + Sync {
     }
 }
 
+/// DIF (Decimation-In-Frequency) butterfly operation.
+///
+/// Used in the *output-ordering* variant of NTT.
+/// This butterfly computes:
+///
+///   output_1 = x1 + x2
+///   output_2 = (x1 - x2) * twiddle
+///
```

Comment:
```suggestion
/// ```text
///   output_1 = x1 + x2
///   output_2 = (x1 - x2) * twiddle
/// ```
```